pkg/apis/aadpodidentity/sort.go

```diff
@@ -0,0 +1,18 @@
+package aadpodidentity
+
+type AzureIdentityBindings []AzureIdentityBinding
+
+func (a AzureIdentityBindings) Len() int {
+	return len(a)
+}
+
+func (a AzureIdentityBindings) Swap(i, j int) {
+	a[i], a[j] = a[j], a[i]
+}
+
+func (a AzureIdentityBindings) Less(i, j int) bool {
```

Comment:
Could we add unit tests for this and the `sort.Sort` with AzureIdentityBindings?